lib/rollout.rb

```diff
@@ -285,6 +285,10 @@ def clear!
     @storage.del(features_key)
   end
 
+  def exists?(feature)
+    !!@storage.get(key(feature))
```

Comment:
Can you please use `exists` instead of `get` for optimal performance? https://redis.io/commands/exists